seatunnel-web-it/README.md

```diff
@@ -0,0 +1,13 @@
+Build seatunnel-web
+./mvnw clean install -DskipTests
+
+Update mysql database details in src/test/resources/application.yml and Run the seatunnel-web-it integration tests
```

Comment:
How about add mysql environment in e2e? We have some example in https://github.com/apache/seatunnel/blob/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-mysql-e2e/src/test/java/org/apache/seatunnel/connectors/seatunnel/cdc/mysql/MysqlCDCIT.java#L107

Reply:
Good suggestion. 
Sure, we can add mysql environment in e2e test cases, but later and in separate PR.